ravendb/documents/operations/connection_string/GetConnectionStringOperation.py

```diff
@@ -0,0 +1,73 @@
+﻿import json
+from typing import Dict, Optional
+
+import requests
+
+from ravendb import RavenCommand, ServerNode
+from ravendb.documents.operations.definitions import MaintenanceOperation
+from ravendb.documents.operations.etl.configuration import RavenConnectionString
+from ravendb.documents.operations.etl.olap import OlapConnectionString
+from ravendb.documents.operations.etl.sql import SqlConnectionString
+from ravendb.serverwide.server_operation_executor import ConnectionStringType
+
+
+class GetConnectionStringsResult:
+    def __init__(
+        self,
+        raven_connection_strings: Dict[str, RavenConnectionString] = None,
+        sql_connection_strings: Dict[str, SqlConnectionString] = None,
+        olap_connection_strings: Dict[str, OlapConnectionString] = None,
+    ):
+        self.raven_connection_strings = raven_connection_strings
+        self.sql_connection_strings = sql_connection_strings
+        self.olap_connection_strings = olap_connection_strings
+
+    def to_json(self) -> Dict:
+        return {
+            "RavenConnectionStrings": self._raven_connection_strings,
+            "SqlConnectionStrings": self._sql_connection_strings,
+            "OlapConnectionStrings": self._olap_connection_strings,
```

Comment:
last thing - shouldn't you call `to_json` on each connection string object from this collection? like this
```suggestion
            "OlapConnectionStrings": [olap_cs.to_json() for olap_cs in self._olap_connection_strings],
```

also in from_json - you need to call from_json on each dict in json_dict["RavenConnectionString"]